api/libproject/environments.rb

```diff
@@ -55,7 +55,7 @@ def env_with_defaults(env, config)
     :tanagra_auth_disable_checks => "false",
     :tanagra_auth_bearer_token => "true",
```

Comment:
Need to change this if you accept the change above.
```suggestion
    : tanagra_auth_gcp_access_token => "true",
```